packages/components/package.json

```diff
@@ -65,18 +69,23 @@
     "@testing-library/react": "^9.3.0",
     "@types/jest": "^24.0.19",
     "@types/jest-axe": "^3.2.0",
+    "autoprefixer": "^10.1.0",
     "babel-loader": "^8.0.6",
+    "babel-plugin-macros": "^3.0.0",
     "babel-plugin-module-resolver": "^4.0.0",
     "jest": "^24.9.0",
     "jest-axe": "^3.2.0",
     "node-sass": "^4.14.1",
+    "postcss": "^8.2.0",
     "react": "^16.10.2",
     "react-docgen-typescript-loader": "^3.3.0",
     "react-dom": "^16.10.2",
     "sass-loader": "^8.0.2",
     "ts-jest": "^24.1.0",
     "ts-loader": "^6.2.0",
-    "typescript": "^4.1.2"
+    "twin.macro": "^2.0.5",
+    "typescript": "^4.1.2",
+    "vue": "^2.6.12"
```

Comment:
I saw warnings that `vue`, `autoprefixer`, and `postcss` were peer dependencies

Reply:
we definitely shouldn't need vue - I'm guessing this is a weird peer dependency that is meant to define what versions it's compatible with. We should try to exclude it even if there are warnings.